website/pages/docs/release-notes/1.5.0.mdx

```diff
@@ -0,0 +1,64 @@
+---
+layout: docs
+page_title: 1.5.0
+sidebar_title: 1.5.0
+description: |-
+  This page contains release notes for Vault 1.5.0.
+---
+
+# Vault 1.5.0
+
+## Vault 1.5 Release Highlights
+
+**Resource Quotas:**  A new set of functionality called Resource Quotas that allows operators to define API quotas which impose usage limits at the system, endpoint, and namespace level (Enterprise Only) in order to ensure applications and users do not overuse Vault’s resources.  The following quota options will be supported:
+* **Lease Count Quotas (Enterprise Only):** Allows operators to specify lease count quotas. If the number of leases in the cluster hits the configured quota limits, additional lease creations will be forbidden for all clients until a lease has been revoked or has expired.
+* **Rate Limit Quotas (All versions):** Allows operators to specify request-per-second quotas. Rate limit quotas are applicable to every node in the Vault cluster, meaning each node will maintain separate counters to enforce rate limits. If the rate limit quota limit is hit on any of the nodes in the Vault cluster, additional requests will be canceled for all clients. 
```

Comment:
I think you could remove the “(All versions)” reference since no enterprise-only reference should imply that.